databrowser/src/domain/datasets/ui/tableView/toolBox/SearchAndFilterToolBoxPanel.vue

```diff
@@ -5,110 +5,113 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
 <template>
-  <ToolBoxPanel>
-    <ToolBoxCard>
-      <ToolBoxCardHeader>
-        <div class="flex items-center gap-2">
-          {{
-            t('datasets.listView.toolBox.searchAndFilter.otherFilters.header')
-          }}
-          <InfoFilter />
-        </div>
-        <ResetAllFilters @reset-all-filters="removeAllFilters" />
-      </ToolBoxCardHeader>
 
-      <ToolBoxCardBody
-        v-for="(filter, index) in tableFilters"
-        :key="`${filter.propertyPath}-${index}`"
-        class="flex flex-col gap-2"
+  <ToolBoxSectionLabel>
+      <span class="mr-4">
+        <span class="text-black">{{props.pagination.pageCount}}</span> <span class="text-dialog">{{ t('datasets.listView.recordsOutOf') }}</span> <span class="text-black">{{ props.pagination.totalItems }}</span>
```

Comment:
No need to write `props. ...` in a Vue template, you can just write e.g. `pagination.pageCount`. The Vue compiler takes care of the rest